src/CosmosDB/CosmosDB/Restore/GetAzCosmosDBMongoDBRestorableCollection.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Management.Automation;
+using System.Web;
+using Microsoft.Azure.Commands.CosmosDB.Helpers;
+using Microsoft.Azure.Management.CosmosDB.Models;
+
+namespace Microsoft.Azure.Commands.CosmosDB
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBMongoDBRestorableCollection", DefaultParameterSetName = NameParameterSet), OutputType(typeof(PSRestorableMongodbCollectionGetResult))]
+    public class GetAzCosmosDBMongoDBRestorableCollection : AzureCosmosDBCmdletBase
+    {
+        [Parameter(Mandatory = true, ParameterSetName = NameParameterSet, HelpMessage = Constants.LocationNameHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string LocationName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = NameParameterSet, HelpMessage = Constants.AccountInstanceIdHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseAccountInstanceId { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = NameParameterSet, HelpMessage = Constants.DatabaseResourceIdHelpMessage)]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseRid { get; set; }
```

Comment:
I recommend using `DatabaseId` directly.

Reply:
Actually this field refers Database Resource Id which is different than Database Id. Used DatabaseRid just to avoid confusion with DatabaseId field.

Reply:
Make sense. How about DatabaseRid -> DatabaseRId to follow Pascal naming?

Reply:
done